common/functions-v2/src/scalars/string.rs

```diff
@@ -39,5 +40,28 @@ pub fn register(registry: &mut FunctionRegistry) {
             Ok(())
         },
     );
+    registry.register_1_arg::<StringType, NumberType<u64>, _, _>(
+        "bit_length",
+        FunctionProperty::default(),
+        |_| None,
+        |val| 8 * val.len() as u64,
+    );
+    registry.register_1_arg::<StringType, NumberType<u64>, _, _>(
+        "octet_length",
+        FunctionProperty::default(),
+        |_| None,
+        |val| val.len() as u64,
+    );
+    registry.register_1_arg::<StringType, NumberType<u64>, _, _>(
+        "char_length",
+        FunctionProperty::default(),
+        |_| None,
+        |val| match std::str::from_utf8(val) {
+            Ok(s) => s.chars().count() as u64,
+            Err(_) => val.len() as u64,
+        },
+    );
     registry.register_aliases("upper", &["ucase"]);
+    registry.register_aliases("octet_length", &["length"]);
+    registry.register_aliases("char_length", &["character_length"]);
```

Comment:
Maybe it's better to move aliases to near the function definition. What do you think?